explorer/src/lib/components/provisioners-list/ProvisionersList.svelte

```diff
@@ -39,39 +39,37 @@
       >{middleEllipsis(
         data.key,
         calculateAdaptiveCharCount(screenWidth, 320, 1024, 4, 30)
-      )}</svelte:fragment
+      ).toUpperCase()}</svelte:fragment
```

Comment:
I would prefer to do this with CSS instead. What would be the best and most consistent way, given the current implementation for rest of the list views? I couldn't see passing a class specifically for this entry, and didn't want to make everything uppercased in the list view, as this would have affected things like the time display, which would be more readable as it is now.

cc: @deuch13